app/services/integration_customers/update_service.rb

```diff
@@ -14,15 +14,13 @@ def call
       return result if integration_customer.type == 'IntegrationCustomers::AnrokCustomer'
       return result.not_found_failure!(resource: 'integration_customer') unless integration_customer
 
-      integration_customer.update!(external_customer_id:) if external_customer_id.present?
+      integration_customer.external_customer_id = external_customer_id if external_customer_id.present?
+      integration_customer.targeted_object = targeted_object if targeted_object.present?
+      integration_customer.save!
 
-      if sync_with_provider
-        integration_customer.subsidiary_id = subsidiary_id if subsidiary_id.present?
-
-        update_result = Integrations::Aggregator::Contacts::UpdateService.call(integration:, integration_customer:)
+      if integration_customer.external_customer_id.present?
```

Comment:
Why don't we store subsidiary_id anymore?

Reply:
Because we don't need to in case of updating a customer. If the customer was succesfully created the subsidiary was saved already.